app/assets/stylesheets/partials/_suggestion-panel.scss

```diff
@@ -0,0 +1,106 @@
+/* Color Variables */
```

Comment:
Non-blocking question (and maybe more for @djanelle-mit than you) - reading this stylesheet, it feels like the long-term plan for many of these variables, mixins, and other styles is to house them more centrally to the application stylesheet setup? Ultimately I think _this_ partial could end up being just lines 27-59?

I don't mind this change as-is for now, but I want to confirm whether I understand the long-term plan correctly.